src/compas_xr/assemblies/assemblies.py

```diff
@@ -0,0 +1,94 @@
+
+import os
+from compas.data import json_dumps,json_loads
```

Comment:
We have configured `isort` to force one-import-per-line, so this should be 
```suggestion
from compas.data import json_dumps
from compas.data import json_loads
```

but you usually don't have to sort this by hand, in VSCode, just `right click -> organize imports`

Reply:
Done :)